# default parameters for the sabertooth 2x60 settings -   PCA9685: 2ms/20ms * 4096

PERIOD_HZ: 50
MOTOR_A_CHANNEL: 0
MOTOR_B_CHANNEL: 1
PWM_BACKWARDS_MIN: 183
PWM_BACKWARDS_MAX: 295
PWM_NEUTRAL_MIN: 298
PWM_NEUTRAL_MAX: 334
PWM_FORWARD_MIN: 336
PWM_FORWARD_MAX: 464
